src/oc_erchef/priv/reindex-opc-organization

```diff
@@ -41,9 +41,14 @@
 %%   2 - Invalid organization name given
 %%
 main(Args) ->
-    init_network(),
-    [Command, OrgInfo, Context] = validate_args(Args),
-    perform(Command, Context, OrgInfo).
+    try
+      init_network(),
+      [Command, OrgInfo, Context] = validate_args(Args),
+      perform(Command, Context, OrgInfo)
+    catch Type:Message ->
+      io:format("~n Caught ~p with the message ~p~n Please try again after restarting the chef server ~n", [Type, Message])
```

Comment:
```suggestion
      io:format("~n Caught ~p with the message ~p~n Please try again after restarting the Chef Infra Server ~n", [Type, Message])
```

Reply:
Could we catch on more specific errors and provide solution accordingly?
- for example:
   - match on the error when the erlang connection cannot happen - explain this happens during the connection process, before reindex takes place and proposed solution.
   - match on the error if unable to bring up an erlang vm because the port is blocked
   - and then keep this as the more generic case.

Reply:
Thanks for the Input! I could not reproduce the blocked port case. Except that, the rest of the changes are updated.